presto-hive/src/main/java/com/facebook/presto/twitter/elephantbird/serde/ThriftSerDe.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.elephantbird.hive.serde;
```

Comment:
That's because the dal-sync was using the elephant-bird's hive serDe. Later, I realize it's not worth to use their serDe class. Since the elephant-bird dependency is not in the hive, as a result I cannot use hive-cli to alter the serDe class. Meanwhile, I don't want to change dal-sync for one more turn of initial backfill, which caused the dal failed all its client's request and some people got paged.